influxql/ast_test.go

```diff
@@ -451,7 +451,7 @@ func TestSelectStatement_IsRawQuerySet(t *testing.T) {
 			isRaw: false,
 		},
 		{
-			stmt:  "select mean(*) from foo group by *",
```

Comment:
`mean(*)` isn't currently valid in our system.  Most of the aggregates don't know how to handle a `*`  and require a field.  With the changes to the validator picking this up earlier now, it surfaced this in the test and had to be fixed.  In the future this restriction will be lifted.